src/Frontend/Core/Engine/Header.php

```diff
@@ -132,7 +132,11 @@ public function addCSS($file, $minify = true, $addTimestamp = null)
         $file = (string) $file;
         $minify = (bool) $minify;
         $addTimestamp = (bool) $addTimestamp;
-        $file = Theme::getPath($file);
+
+        // get file path
+        if (mb_substr($file, 0, 4) != 'http') {
```

Comment:
I disagree with your argument. To me, `strpos` is 'is the http string present at the start of the string' while `substr` says 'are the 4 first letters of the string http'? Personally I find the latter a tad more logical and since @jessedobbelaere already implemented it like this, I'd just like to merge it :)